src/components/SummaryBox/SummaryBox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import { SummaryBox } from './SummaryBox'
+
+export default {
+  title: 'Components/SummaryBox',
+  component: SummaryBox,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+### USWDS 2.0 SummaryBox component
+
+Source: https://designsystem.digital.gov/components/summary-box
+`,
+      },
+    },
+  },
+}
+
+const summaryBoxContent = (
+  <div className="usa-summary-box__text">
```

Comment:
I would personally include this div as a part of `SummaryBox` such that this is the div wrapping `children`. 

That would still allow for content (`ul`s, `p`s, etc.) to be passed in. 

What are your thoughts?

Reply:
I think it makes sense to include it, I probably left it as part of the Storybook example as part of the MVP. Moving it over now.